src/preprocess.py

```diff
@@ -616,6 +619,17 @@ def add_openai_bpe_vocab(vocab, namespace='openai_bpe'):
     vocab.add_token_to_namespace(utils.EOS_TOK, namespace)
 
 
+def add_wsj_vocab(vocab, data_dir, namespace='tokens'):
```

Comment:
This shouldn't be here...

Reply:
I think you messed something up when merging changes to this file.

Reply:
I think this was because this branch was behind the master.... should be fixed now.